guibot/config.py

```diff
@@ -501,6 +501,55 @@ class GlobalConfig(object, metaclass=GlobalConfig):
     pass
 
 
+class TemporaryConfig(GlobalConfig):
+    """
```

Comment:
Hmmm, is there much use in inheriting from `GlobalConfig` here? It seems that all operations below refer to this class directly.